src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -311,6 +312,20 @@ void onMouseEnterGameStatus() {
   }
 
   @FXML
+  void onMouseEnterUsername() {
+    if (playerInfoBean.getChatOnly()) {
+      return;
+    } else {
+      PlayerCardTooltipController playerCardTooltipController = applicationContext.getBean(PlayerCardTooltipController.class);
+      playerCardTooltipController.setPlayer(playerInfoBean);
+
+      Tooltip statusTooltip = new Tooltip();
+      statusTooltip.setGraphic(playerCardTooltipController.getRoot());
+      Tooltip.install(usernameLabel, statusTooltip);
+    }
```

Comment:
This installs a tooltip on every mouse-over, even if it has already been installed.

Also, there already is a tooltip installed. See `configureTooltip()`